certified-connectors/Azure Machine Learning/apiProperties.json

```diff
@@ -0,0 +1,68 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "region": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "required": "true"
+          },
+          "description": "Specify the region of your pipeline",
+          "displayName": "Region",
+          "tooltip": "Provide your region. Eg: centralus, etc."
+        }
+      },
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aadcertificate",
+          "properties": {
+            "IsFirstParty": "True"
+          },
+          "clientId": "<<Enter client id>>",
+          "scopes": [],
+          "redirectMode": "Global",
+          "customParameters": {
+            "loginUri": {
+              "value": "https://login.windows.net"
+            },
+            "tenantId": {
+              "value": "common"
+            },
+            "resourceUri": {
+              "value": "https://management.azure.com/"
+            }
+          }
+        }
+      },
+      "token:TenantId": {
+        "type": "string",
+        "metadata": {
+          "sourceType": "AzureActiveDirectoryTenant"
+        },
+        "uiDefinition": {
+          "constraints": {
+            "required": "false",
+            "hidden": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#FFFFFE",

```

Comment:
@yashodeepm if it's possible, could you please change the background color to light grey, e.g. `#FEFEFE`? This property should match icon's background color, and the icon should not have white or transparent background.